Tests/FrameworkTests/EmptyFileTests.swift

```diff
@@ -0,0 +1,75 @@
+import SwiftLintFramework
+import XCTest
+
+private protocol BooleanConstant {
+    static var boolValue: Bool { get }
+}
+
+private struct True: BooleanConstant {
+    static var boolValue: Bool { true }
+}
+
+private struct False: BooleanConstant {
+    static var boolValue: Bool { false }
+}
+
+private struct RuleWithLintEmptyFilesMock<ShouldLintEmptyFiles: BooleanConstant>: CorrectableRule {
```

Comment:
A short name with this `ShouldLintEmptyFiles` parameter should reveal the rules purpose sufficiently well:

```suggestion
private struct RuleMock<ShouldLintEmptyFiles: BooleanConstant>: CorrectableRule {
```